src/librustc/infer/error_reporting/mod.rs

```diff
@@ -847,7 +847,7 @@ impl<'a, 'gcx, 'tcx> InferCtxt<'a, 'gcx, 'tcx> {
                                                "{} may not live long enough",
                                                labeled_user_string);
                 err.help(&format!("consider adding an explicit lifetime \
-                                   bound `{}: 'static`...",
+                                   bound `{}: 'static + `...",
```

Comment:
Oh I see @nikomatsakis. Yes, my point was printing `T: 'static +` but I was not aware of the other usage.

Ok now, do you want me to make any more changes to this PR?